ui/src/alert-configuration/ConfigureAlerting.test.tsx

```diff
@@ -0,0 +1,1275 @@
+      'utf8',
+    );
+    server.use(
+      rest.get(
+        `http://localhost/api/kubernetes/api/v1/namespaces/metalk8s-monitoring/configmaps/metalk8s-alertmanager-config`,
+        (req, res, ctx) => {
+          const test = {
+            kind: 'ConfigMap',
+            apiVersion: 'v1',
+            metadata: {
+              name: 'metalk8s-alertmanager-config',
+              namespace: 'metalk8s-monitoring',
+              uid: 'xxx-yyy-zzz',
+              resourceVersion: '0',
+              creationTimestamp: '2023-03-23T16:45:41Z',
+              labels: {},
+              managedFields: [],
+            },
+            data: { 'config.yaml': configYaml },
+          };
+          return res(ctx.json(test));
+        },
+      ),
+    );
+    await commonSetup();
+
+    expect(selectors.enableConfiguration()).toBeChecked();
+    expect(selectors.host()).toHaveValue('smtp4dev.default.svc.cluster.local');
+    expect(selectors.port()).toHaveValue(25);
+    expect(selectors.enableTls()).toBeChecked();
+    expect(selectors.authSelect.label()).toHaveTextContent('NO AUTHENTICATION');
+
+    expect(selectors.sender()).toHaveValue('admin@scality.com');
+    expect(selectors.recipient()).toHaveValue(
+      'Renard <renard@scality.com>, Chat <chat@scality.com>, Lapin <lapin@scality.com>',
+    );
+    expect(selectors.receiveResolved()).toBeChecked();
+  });
+
+  it('render form values when the form is defined (LOGIN)', async () => {
+    const configYaml = fs.readFileSync(
+      path.join(__dirname, './test-yaml/simple-login.yaml'),
+      'utf8',
+    );
+    server.use(
+      rest.get(
+        `http://localhost/api/kubernetes/api/v1/namespaces/metalk8s-monitoring/configmaps/metalk8s-alertmanager-config`,
+        (req, res, ctx) => {
+          const test = {
+            kind: 'ConfigMap',
+            apiVersion: 'v1',
+            metadata: {
+              name: 'metalk8s-alertmanager-config',
+              namespace: 'metalk8s-monitoring',
+              uid: 'xxx-yyy-zzz',
+              resourceVersion: '0',
+              creationTimestamp: '2023-03-23T16:45:41Z',
+              labels: {},
+              managedFields: [],
+            },
+            data: { 'config.yaml': configYaml },
+          };
+          return res(ctx.json(test));
+        },
+      ),
+    );
+    await commonSetup();
+
+    expect(selectors.enableConfiguration()).toBeChecked();
+    expect(selectors.host()).toHaveValue('smtp4dev.default.svc.cluster.local');
+    expect(selectors.port()).toHaveValue(25);
+    expect(selectors.enableTls()).toBeChecked();
+    expect(selectors.authSelect.label()).toHaveTextContent('LOGIN');
+    expect(selectors.username()).toHaveValue('Renard');
+    expect(selectors.password()).toHaveValue('Renard Password');
+
+    expect(selectors.sender()).toHaveValue('admin@scality.com');
+    expect(selectors.recipient()).toHaveValue(
+      'Renard <renard@scality.com>, Chat <chat@scality.com>, Lapin <lapin@scality.com>',
```

Comment:
nit: 🤣 !! Maybe we should still translate them in english but that super fine !!

Reply:
I will keep it in French haha. It's just for fun anyway. 😂